### PR TITLE
add support for output format in json or yaml

### PR DESCRIPTION
**Pull request by rajatjindal**
It would be nice to be able to have history command print output in json or yaml format that can be consumed by other tools.

the PR add support for yaml and json formats and keeps existing behavior as default.

### CODE REVIEW

**Comment by rajatjindal** [comment]
sample output here: https://gist.github.com/rajatjindal/9bdb83db13cd2995484d4e9f3bb510e4

**Comment by bacongobbler** [comment]
would you mind running `make docs`?

> helm docs are out of date. Please run "make docs"

**Comment by rajatjindal** [comment]
@bacongobbler updated now. sorry missed it somehow earlier.

**Comment by rajatjindal** [comment]
@bacongobbler can you please review the changes

**Comment by rajatjindal** [comment]
@bacongobbler gentle reminder to review the changes

File: cmd/helm/history.go
Comment (bacongobbler) [inline cmd/helm/history.go]: can we change the docstring to "prints the output in the specified format (json|table|yaml)"? cobra will automatically display the default value, so it's kinda redundant:

```
><> ./bin/helm help history | grep -- --output
  -o, --output string        prints the output in the specified format ('json' or 'yaml' or defaults to 'table') (default "table")
```

**Review by bacongobbler — CHANGES_REQUESTED** [review]
one minor doc change, but the rest looks good to me!

**Comment by rajatjindal** [comment]
thanks for the review @bacongobbler I've updated the doc as recommended

**Review by bacongobbler — APPROVED** [review]

**Comment by bacongobbler** [comment]
needs one extra sign-off from another maintainer since it's a larger PR :)

**Comment by rajatjindal** [comment]
thanks @bacongobbler 

pinging @thomastaylor312 @technosophos  may be one of them can find sometime to approve this PR as well.

between our team just did an internal demo using helm/kubernetes and everyone was like wow !!. 

thanks for the great work.

**Comment by thomastaylor312** [comment]
@rajatjindal I'll do another review tomorrow!

**Comment by rajatjindal** [comment]
gentle reminder @thomastaylor312

**Comment by rajatjindal** [comment]
gentle reminder @thomastaylor312

**Review by thomastaylor312 — APPROVED** [review]
So sorry for the delay! The last part of my week was crazy. This is good to go for merge @bacongobbler. Thanks again for adding this @rajatjindal and I appreciate the patience (and reminders)

**Pull request merged**
